src/Compilers/CSharp/Portable/Utilities/ValueSetFactory.EqualableValueTC.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    internal static partial class ValueSetFactory
+    {
+        /// <summary>
+        /// A type class for values (of type <typeparamref name="T"/>) that can be directly compared for equality 
+        /// using <see cref="System.Object.Equals(object?, object?)"/>.
+        /// </summary>
+        private interface EqualableValueTC<T>
```

Comment:
>Equalable [](start = 26, length = 9)

Consider naming `EquatableValueTC` instead.

Reply:
Will rename to IEquatableValueTC in the next PR.

---
In reply to: [383453603](https://github.com/dotnet/roslyn/pull/41503#discussion_r383453603) [](ancestors = 383453603)